tests/test_photodiode.py

```diff
@@ -0,0 +1,191 @@
+import unittest
+from unittest.mock import MagicMock, call
+import sys
+
+mock_psychopy = MagicMock()
+sys.modules["psychopy"] = mock_psychopy
+sys.modules["psychopy.visual"] = mock_psychopy.visual
+sys.modules["psychopy.core"] = mock_psychopy.core
+
+mock_serial = MagicMock()
+sys.modules["serial"] = mock_serial
+
+mock_pylsl = MagicMock()
+sys.modules["pylsl"] = mock_pylsl
+
+import tools.experiment.photodiode as pd
```

Comment:
This is typically reserved for pandas. Perhaps something more explicit `photodiode_exp`?